client/components/ItemDetails.jsx

```diff
@@ -1,12 +1,31 @@
 import React from 'react';
 
-const ItemDetails = (props) => (
-  <div className="Item-Details">
-    <ul className="Item-Details-List">
-      {props.details.map((item, key) => (<li key={key}>{item}</li>))}
-    </ul>
-  </div>
-);
+class ItemDetails extends React.Component {
```

Comment:
I'm also curious as to your thought process for converting all these functional components into class components. It doesn't appear that you are using state anywhere in them.

Reply:
Gotcha. I think you can define propTypes on functional components too. You just have to require the prop-types npm module in each component file, and then I think it is just the same process as for class components.